apps/safe-settings/safe-settings/safe-settings.yaml

```diff
@@ -17,12 +17,21 @@ spec:
   releaseName: safe-settings
   values:
     appEnv:
+      ADMIN_REPO:
+        valueFrom:
+          secretKeyRef:
+            name: app-env
+            value: ADMIN_REPO
       APP_ID:
         valueFrom:
           secretKeyRef:
             name: app-env
             value: APP_ID
-      CRON: "0 */1 * * *"
```

Comment:
why are most of these values secret? a cron schedule and app id aren't sensitive and it just makes it harder to see what's going on here.

Reply:
It doesn't seem to like mixing and matching secret values with non-secret ones

Reply:
i guess worth a try but it shouldn't matter